apps/mobile/src/stacks/Dashboard/index.tsx

```diff
@@ -13,8 +17,15 @@ const Tab = createBottomTabNavigator<DashboardParamList>();
 
 export const DashboardStack = () => {
 	useNotificationPermissionRequest();
+	const insets = useSafeAreaInsets();
 
 	const screenOptions: BottomTabNavigationOptions = {
+		header({ route }) {
+			return <Header topInset={insets.top} title={route.name} />;
+		},
+	};
+
+	const noHeader = {
 		headerShown: false,
 	};
```

Comment:
put this on separate file, e.g under util

Reply:
please recommend me a name for that file